addons/sale_timesheet/views/hr_timesheet_views.xml

```diff
@@ -173,58 +173,83 @@
-        <field name="res_model">account.analytic.line</field>
-        <field name="view_mode">tree,form,pivot,graph,kanban</field>
+        <field name="res_model">timesheets.analysis.report</field>
         <field name="domain">[('project_id', '!=', False)]</field>
-        <field name="context">{
-            'search_default_groupby_timesheet_invoice_type': 1,
-            'pivot_row_groupby': ['date:month'],
-        }</field>
         <field name="help" type="html">
             <p class="o_view_nocontent_smiling_face">
                 No data yet!
             </p>
             <p>Review your timesheets by billing type and make sure your time is billable.</p>
         </field>
         <field name="search_view_id" ref="hr_timesheet.hr_timesheet_line_search"/>
+        <field name="view_mode">pivot,graph</field>
     </record>
 
-    <record id="timesheet_action_view_report_by_billing_rate_form" model="ir.actions.act_window.view">
-        <field name="sequence" eval="10"/>
-        <field name="view_mode">form</field>
-        <field name="view_id" ref="hr_timesheet.hr_timesheet_line_form"/>
-        <field name="act_window_id" ref="timesheet_action_billing_report"/>
-    </record>
+    <record id="timesheets_analysis_report_pivot_invoice_type" model="ir.ui.view">
+            <field name="name">timesheets.analysis.report.pivot</field>
+            <field name="model">timesheets.analysis.report</field>
+            <field name="arch" type="xml">
+                <pivot string="Timesheets Analysis" sample="1" disable_linking="True">
+                    <field name="amount" string="Timesheet Costs"/>
+                    <field name="unit_amount" type="measure" widget="timesheet_uom"/>
+                    <field name="timesheet_invoice_type" type="row"/>
+                    <field name="date" interval="month" type="col"/>
+                </pivot>
+            </field>
+        </record>
 
-    <record id="timesheet_action_view_report_by_billing_rate_kanban" model="ir.actions.act_window.view">
-        <field name="sequence" eval="10"/>
-        <field name="view_mode">kanban</field>
-        <field name="view_id" ref="hr_timesheet.view_kanban_account_analytic_line"/>
-        <field name="act_window_id" ref="timesheet_action_billing_report"/>
-    </record>
+        <record id="timesheets_analysis_report_graph_invoice_type" model="ir.ui.view">
+            <field name="name">timesheets.analysis.report.graph</field>
+            <field name="model">timesheets.analysis.report</field>
+            <field name="arch" type="xml">
+                <graph string="Timesheets" sample="1" js_class="hr_timesheet_graphview" disable_linking="True">
+                    <field name="amount" string="Timesheet Costs"/>
+                    <field name="unit_amount" type="measure" widget="timesheet_uom"/>
+                    <field name="timesheet_invoice_type" type="row"/>
+                </graph>
+            </field>
+        </record>
```

Comment:
To move in the report folder and at the top of the file to have all the views definition at the same place. 😇 
Thanks in advance! 🙂